core/src/main/scala/org/http4s/headers/Authorization.scala

```diff
@@ -7,12 +7,38 @@
 package org.http4s
 package headers
 
-import org.http4s.parser.HttpHeaderParser
-import org.http4s.util.Writer
+import cats.syntax.all._
+import org.http4s.util.{CaseInsensitiveString, Writer}
+import cats.parse._
 
 object Authorization extends HeaderKey.Internal[Authorization] with HeaderKey.Singleton {
+  //https://tools.ietf.org/html/rfc7235#section-4.2
+  private val parser: Parser1[Authorization] = {
+    import Parser.char
+    import cats.parse.Rfc5234.sp
+
+    import org.http4s.internal.parsing.Rfc7230.{headerRep1, ows, quotedString, token}
+    import org.http4s.internal.parsing.Rfc7235.token68
+
+    //auth-scheme = token
+    val scheme = token.map(CaseInsensitiveString(_))
+    //val authParamValue = token.orElse(quotedString)
+    // auth-param = token BWS "=" BWS ( token / quoted-string )
+    val authParam: Parser1[(String, String)] =
+      (token <* (ows ~ char('=').void ~ ows)) ~ quotedString
```

Comment:
missing token here.

Reply:
Should we alias `bws` to `ows`?  I had to remind myself what `bws` is.